README.md

```diff
@@ -54,27 +54,26 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 ## Install
 
-Scripts for building and running CoatRack are available in the root folder of the code base. To build CoatRack, use:
+Go to the docker-compose-setup folder and execute:
```

Comment:
Before this I would suggest to add a short "pre-conditions" list for the installation, e.g. a short bullet point list including java 11 or later, maven 3.? or later, docker and docker compose...

Also it would be good to add a short explanation of what the init script does (maybe also as bullet points): build coatrack, create a local docker image, create volumes, etc.